test/contextualLifecycleRule.spec.ts

```diff
@@ -150,6 +150,170 @@ describe(ruleName, () => {
       });
     });
 
+    describe('failure', () => {
```

Comment:
```suggestion

```

We already have a _failure_ **describe** block. You can put the `NgModule` block into it.